tests/JasmineSidetreeErrorValidator.ts

```diff
@@ -10,15 +10,21 @@ export default class JasmineSidetreeErrorValidator {
    *
    * @param functionToExecute The function to execute.
    * @param expectedErrorCode The expected error code.
+   * @param expectedContainedStringInMessage The string expected to be part of (need not be full error message) the error message.
    */
-  public static expectSidetreeErrorToBeThrown (functionToExecute: () => any, expectedErrorCode: string): void {
+  public static expectSidetreeErrorToBeThrown (functionToExecute: () => any, expectedErrorCode: string, expectedContainedStringInMessage?: string): void {
     let validated: boolean = false;
 
     try {
       functionToExecute();
     } catch (e) {
       if (e instanceof SidetreeError) {
         expect(e.code).toEqual(expectedErrorCode);
+
+        if (expectedContainedStringInMessage !== undefined) {

```

Comment:
Yaaaay, I like this.

Reply:
Do you also like `util:generate-error-codes`? =P